src/View/mainpage/DrawROIWindow/SelectROIPopUp.py

```diff
@@ -0,0 +1,115 @@
+import csv
+import platform
+
+from PySide6 import QtCore, QtGui
+from PySide6.QtWidgets import QDialog, QLabel, QLineEdit, QWidget, QPushButton, QHBoxLayout, QListWidget, QVBoxLayout
```

Comment:
Line too long for PEP 8 compliance. Also lines:
10-14, 34, & 73

Also, class and functions have no docstrings.